Pod/Classes/JSSAlertView.swift

```diff
@@ -190,10 +190,10 @@ public class JSSAlertView: UIViewController {
 		// position text
 		if self.textView != nil {
 			let textString = textView.text! as NSString
-			let textAttr = [NSFontAttributeName:textView.font as! AnyObject]
```

Comment:
@tiferrei as mentioned, the problem seems to be on this line! I'm not advanced on Swift and I couldn't figure out whats's wrong. Setting textAttr to static attributes fixed the problem.
Tks